STYLE_GUIDE.md

```diff
@@ -0,0 +1,179 @@
+# Style Guide
+
+In datarockets we enforce a community [Ruby Style Guide](https://github.com/rubocop-hq/ruby-style-guide).
+
+This is a small list of differences which we have when compared with community style guide:
```

Comment:
without `when `

Reply:
`compared to`

Reply:
@AleksSenkou in this case, we got 

```
This is a small list of differences which we have compared to the community style guide
```

and this isn't which I meant before. 

What about?

```
This is a small list of differences which we have after comparing to the community style guide
```